cabot/cabotapp/models.py

```diff
@@ -141,6 +141,11 @@ class Meta:
         help_text='Gist, Hackpad or Refheap js embed with recovery instructions e.g. '
                   'https://you.hackpad.com/some_document.js'
     )
+    hipchat_room_id = models.IntegerField(
```

Comment:
The model can be
```
HipchatInstance
   name (unique)
   server_url
   api_key
```

The string representation of the instance can be Hipchat(<name>)

In the dropdown, we show Hipchat(<name>) which should be good enough.. What do you think?